web/src/constants.ts

```diff
@@ -28,6 +28,7 @@ export const codeMessage = {
   422: '当创建一个对象时，发生一个验证错误。',
   500: '服务器发生错误，请检查服务器。',
   502: '网关错误。',
+
```

Comment:
why change this?

Reply:
From that i was just testing husky, it will not harm any code

Reply:
Yes, but please don't make unnecessary changes.